src/voucherify-integration/emporixApi.js

```diff
@@ -0,0 +1,154 @@
+import { getEmporixAPIAccessToken } from './getEmporixAPIAccessToken'
```

Comment:
Definitely, you can rethink how looks the architecture of your code. You have a service that uses `getEmporixAPIAccessToken` which inside uses local storage for cache 🥶

Reply:
that is why noone shall do server on a frontend